vtest/gen

```diff
@@ -156,8 +156,8 @@ echo "   <link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\">" >> $F
 echo "  </head>" >> $F
 echo "  <body>" >> $F
 echo "    <div id=\"topbar\">" >> $F
-echo "      <span>Current</span>" >> $F
-echo "      <span>Reference</span>" >> $F
+echo "      <span>After PR</span>" >> $F
+echo "      <span>Before PR</span>" >> $F
```

Comment:
Maybe just "Before" and "After" are better? or ""change" rather than "PR"? Not sure whether vtests are (or should be) used only on PRs.